api/base/filters.py

```diff
@@ -215,14 +215,22 @@ def parse_query_params(self, query_params):
                         query.get(key).update({
                             field_name: self._parse_date_param(field, source_field_name, op, value)
                         })
-                    elif not isinstance(value, int) and (source_field_name in ['_id', 'root']):
+                    elif not isinstance(value, int) and (source_field_name == '_id'):
                         query.get(key).update({
                             field_name: {
                                 'op': 'in',
                                 'value': self.bulk_get_values(value, field),
                                 'source_field_name': source_field_name
                             }
                         })
+                    elif not isinstance(value, int) and (source_field_name == 'root'):
```

Comment:
One question: by combining both cases  `'_id'`,` 'root'`, we also allow` [id][ne]` which is currently handled as just `[id]`.